packages/webapp/src/routes/combined-case/match-habitats.js

```diff
@@ -0,0 +1,70 @@
+import constants from '../../utils/constants.js'
+import getHabitatType from '../../utils/getHabitatType.js'
+import mockMetricData from './mock-metric-data.js'
+
+const habitatKeys = ['d2', 'd3', 'e2', 'e3', 'f2', 'f3']
+
+export const getNumberOfPages = (data) => habitatKeys.reduce((acc, key) => acc + data[key].slice(0, -1).length, 0)
```

Comment:
Just a note, nothing to change.

Good for now, but this might change as not all allocation habitats will have something match against. But we need to work that out later when we do the habitat matching logic